doc/sphinx/how-to/how-to-integrate-a-custom-wayland-protocol.md

```diff
@@ -0,0 +1,137 @@
+---
+discourse: 11993
+---
+
+# How to integrate a custom Wayland protocol
+
+With the [Mir 1.3 release](https://discourse.ubuntu.com/t/mir-1-3-0-release/11638/2)
+we smoothed the way for shells to add custom Wayland protocols to their Mir-based
+compositors.
+
+First, have a Wayland protocol XML on hand. As an example, we will use
+the ["ext-workspace-v1"](https://wayland.app/protocols/ext-workspace-v1)
+protocol.
+
+Next, install the `mirwayland` library. On Debian and its derivatives, this
+can be installed from the mir-team ppa:
+
+```sh
+sudo add-apt-repository --update ppa:mir-team/release
+sudo apt install libmirwayland-dev
+```
```

Comment:
It seems odd to be _adding the PPA_ here: I think we should assume that a developer reaching this point has already installed Mir from somewhere. (Covered in write-your-first-wayland-compositor.md)